src/Host.cpp

```diff
@@ -2249,7 +2297,7 @@ QString Host::getPackageConfig(const QString& luaConfig, bool isModule)
     }
 
     lua_State* L = luaL_newstate();
-    luaL_openlibs(L);
+    setupSandboxedLuaState(L);
```

Comment:
I'm not playing in my baliwick here but from what I've read `luaL_loadString(...)` uses `loadString(...)` - will it still be able to ***after*** the latter gets excised from the `Lua_State`?